open-company/open-company-web
# Fix org save error

Bug: login as admin, open org settings panel, change org name, hit save: :boom: error notification but the operation actually worked.

To test:
- verify the above situation is failing on mainline
- switch to this PR
- repeat the steps above
- [x] worked? Good
- now update org avatar
- add email domain
- remove email domain
- [x] everything fine?
- notifications are always inline with what is happening?
